models/sessionization/segment_web_page_views__sessionized.sql

```diff
@@ -93,7 +93,20 @@ new_sessions as (
         case
             when period_of_inactivity <= {{var('segment_inactivity_cutoff')}} then 0
             else 1
-        end as new_session
+        end as new_session_inactivity,
+        case
+            when previous_tstamp::DATE != tstamp::DATE then 1
+            else 0
+        end as new_session_midnight,
+        case
+           when referrer_host ilike '%beautypiehelp.zendesk%' or referrer_host = 'beautypie.com' then 0
```

Comment:
I feel like these need to be params if this is a pkg...

Reply:
or like, does this need to be in the pkg even?

Reply:
or do we even bother keeping the pkg now...

Reply:
honestly we probably can just move this into ds-dbt 😂  kinda like the facebook package all over again at this point.

Reply:
i can already see all the sqlfluff isses if we move it over lol.